src/cmds/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cd /opt/api
+
+# The entrypoint for the Postscript Homework API
+./cmds/wait-for-it.sh db:5432 -s -- printf "Database Successfully Started\n"
```

Comment:
I get a "permission denied", although this is probably just a problem with my setup
```
> docker run sbubby
/opt/api/cmds/entrypoint.sh: line 6: ./cmds/wait-for-it.sh: permission denied
```

Reply:
Did you install docker as admin?

Reply:
I installed it through the command line with sudo

Reply:
wait, so, entrypoint.sh actually executes, you just need to have the ENTRYPOINT be bash in order to not get this perm error?